types/sentry.d.ts

```diff
@@ -44,19 +39,11 @@ interface TracingOptions {
      * Based on https://vuejs.org/v2/api/#Options-Lifecycle-Hooks
      */
     hooks: Operation[]
-}
-
-export interface TracingConfiguration {
-    tracesSampleRate?: number
-    vueOptions?: {
-        tracing?: boolean
-        tracingOptions?: Partial<TracingOptions>
-    }
-    browserOptions?: Partial<BrowserTracing['options']>
+    tracePropagationTargets: (string|RegExp)[]
```

Comment:
Various issues with the `TracingConfiguration` interface.
- properties should be optional
- `browserTracing` is missing
- `tracePropagationTargets` belongs in `browserTracing`

Reply:
addressed in f6e67a8f39107583ea52da6e90dfd17158789e81